config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  namespace :admin do
+    get "redact_files/new"
+  end
```

Comment:
Do we need both these lines and line 41 below since it is not an AR-backed resource?

Reply:
@littleforest good catch! We only need this in line 41